dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2216,37 +2470,36 @@
       "x-ms-discriminator-value": "LengthResolution",
       "properties": {
         "unit": {
-          "type": "string",
-          "enum": [
-            "Unspecified",
-            "Kilometer",
-            "Hectometer",
-            "Decameter",
-            "Meter",
-            "Decimeter",
-            "Centimeter",
-            "Millimeter",
-            "Micrometer",
-            "Nanometer",
-            "Picometer",
-            "Mile",
-            "Yard",
-            "Inch",
-            "Foot",
-            "LightYear",
-            "Pt"
-          ],
-          "x-ms-enum": {
-            "name": "LengthUnit",
-            "modelAsString": true
-          },
-          "description": "The length Unit of measurement"
+          "$ref": "#/definitions/LengthUnit"
         }
       },
       "required": [
         "unit"
       ]
     },
+    "InformationUnit": {
```

Comment:
These descriptions at least require a description. For example, is a megabyte 1024 kilobytes or 1000 kilobytes. Or could you add mibabytes and friends? Or is that what the megabit is supposed to be? That's not an industry term I've heard, though I suspect those are literally 1024*1 bits, right?

All these questions mean it should at least be described. People will have different assumptions.

Reply:
it doesn't matter to us what the definition is, as we will do no conversion but added a description saying that it could mean both.